apisix/plugins/grpc-transcode/util.lua

```diff
@@ -147,6 +147,22 @@ function _M.map_message(field, default_values, request_table)
         if ty ~= "enum" and field_type:sub(1, 1) == "." then
             if request_table[name] == nil then
                 sub = default_values and default_values[name]
+            elseif core.table.isarray(request_table[name]) then
+                local sub_array = core.table.new(#request_table[name],0)
+                for i,value in ipairs(request_table[name]) do
```

Comment:
```suggestion
                for i, value in ipairs(request_table[name]) do
```

Reply:
I have modified and submitted，but  the CI not pass